identity_iota_core/scripts/publish_identity_package.sh

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 script_dir=$(dirname $0)
-package_dir=$script_dir/../packages/identity_iota
+package_dir=$script_dir/../packages/iota_identity
```

Comment:
Just seeing this update, should we update these as well?

- cache file `/tmp/identity_iota_pkg_id.txt` (grpc bindings)
- env variable `DENTITY_IOTA_PKG_ID`, found some older references in
  - grpc bindings
  - examples
  - identity_iota_core e2e tests, `common` helper

Reply:
Yeah although I rather not do that in this PR